### PR TITLE
New release

### PR DESCRIPTION
**Pull request by Janamou**
Changes: https://firebase.google.com/support/release-notes/js#3.8.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+## 3.0.3
```
Comment (kevmoo) [inline CHANGELOG.md]: Since there are new APIs, we should call this `3.1.0`

Comment (Janamou) [inline CHANGELOG.md]: @kevmoo two tests added, the other methods are for OAuth providers only.

Version updated.

**Comment by Janamou** [comment]
Hi @kevmoo, I added also new changes from yesterday's 3.9.0 update: https://firebase.google.com/support/release-notes/js

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+## 3.1.0
+
+* Update to the latest Firebase `3.9.0`:
+    * User's `link` method deprecated in favor of `linkWithCredential`.
+    * User's `reauthenticate` method deprecated in favor of `reauthenticateWithCredential`.
+    * New methods in the `auth` library.
```
Comment (kevmoo) [inline CHANGELOG.md]: Describe the new ones here?

Comment (Janamou) [inline CHANGELOG.md]: @kevmoo done, more detailed changelog.
Thx!

**Comment by kevmoo** [comment]
@Janamou uh oh - https://travis-ci.org/firebase/firebase-dart/jobs/226444434

**Comment by Janamou** [comment]
@kevmoo I am sorry, fixed now!

**Comment by kevmoo** [comment]
No apologies! That's what travis is for!

**Comment by kevmoo** [comment]
https://github.com/firebase/firebase-dart/releases/tag/v3.10

Published!

**Pull request merged**